testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/authz/EntitlementAPITest.java

```diff
@@ -164,6 +176,65 @@ public void testRptRequestWithResourceNamePairwise() {
         testRptRequestWithResourceName(PAIRWISE_AUTHZ_CLIENT_CONFIG);
     }
 
+    @Test
+    public void testInvalidRequestWithClaimsFromConfidentialClient() throws IOException {
```

Comment:
I am not sure I fully understand this test. The purpose is just to check whether adding claim won't mess up authorization (if yes then it is ok)? Because it looks like the claim is never used in authorization process.

Reply:
Yeah, this is basically checking whether or not passing a claim token is failing when parsing the token in the server.